src/NuGet.Core/NuGet.Configuration/NuGet.Configuration.csproj

```
@@ -5,24 +5,24 @@
   <PropertyGroup>
     <Description>NuGet's configuration settings implementation.</Description>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
-    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworksLibrary);net45</TargetFrameworks>
```

Avoid `net45` TFM in source-build mode:

```suggestion
    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
    <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' != 'true'">$(TargetFrameworks);net45</TargetFrameworks>
```

---

fixed in next iteration.